roles/zuul/meta/main.yml

```diff
@@ -1,2 +1,12 @@
 dependencies:
-  - pip
+  - role: python-app
+    name: zuul
+    python_pipdeps:
+      - name: jenkins-job-builder
+        version: '1.6.1'
+      - name: pyzmq
+      - name: ansible
+      - name: statsd
+    git_repo: "{{ zuul_git_repo_url }}"
+    git_version: "{{ zuul_git_branch }}"
+    python_notify: Restart zuul
```

Comment:
Also make sure that `NOTIFIED HANDLER` is actually followed up with that handler running. Ansible may say that for every thing on a notify, even if there is no matching handler for it. I just don't remember the fine details here.